its/src/test/java/com/sonar/it/csharp/MetricsIncludeHeaderCommentTest.java

```diff
@@ -52,9 +52,10 @@ public class MetricsIncludeHeaderCommentTest {
 
   public static TemporaryFolder temp = TestUtils.createTempFolder();
 
-  private static final String PROJECT = "MetricsTest";
-  private static final String DIRECTORY = "MetricsTest:foo";
-  private static final String FILE = "MetricsTest:foo/Class1.cs";
+  private static final String PROJECT_DIR = "MetricsTest";
```

Comment:
This class collides with `MetricsTest`, so we need to use different project key (but same project dir)